### ISSUE TITLE
Insufficient detail provided for placement of nuget package in Overrides folder.

### ISSUE THREAD

**Issue by Noemata** [tristanlabelle/swift-winui-sample#2]
I was not able to resolve requisite placement of nuget package in the overrides folder.

Renaming appears to be required with particular folder structure and use of *.nuspec and and *.sh512 file.

Need more detail please.

**Comment by tristanlabelle** [comment]
1. Build `swift-winrt` using CMake (simpler)
1. Build the nuget package from `swift-winrt` using `powershell Generator\Create-NuGetPackage.ps1 -NativeExe build\debug\Generator\SwiftWinRT.exe -Version 0.0.0 -OutputPath .\TristanLabelle.SwiftWinRT.0.0.0.nupkg`
1. Copy `TristanLabelle.SwiftWinRT.0.0.0.nupkg` to `Dependencies/NuGet/Overrides` in your checkout of this repo
1. Delete `Dependencies/NuGet/packages.lock.json` in your checkout of this repo
1. Build this repo

It's not documented because this repo is not ready and that's now how things are going to work when I'm done. I'll be publishing SwiftWinRT to nuget.org and this project will download it from there, but I'm iterating locally to get it to work first.

Again, this is all very experimental. I have barely run more than a hello world app with my swift-winrt project.

**Comment by Noemata** [comment]
I did the above, except I've started tweaking this project so it matches the version number of the nugget package generated by "Create-NuGetPackage.ps1".  There, it's "0.0.0.1".  I've just started in again this morning so I'll have to reconfirm some bits.  There's another place in this project where that nugget version number is referenced.  I apologize for making assumptions about your projects.  I assumed these were newer versions of company projects.  It's quite polished over all, so I thought these were production grade.  I prefer the design here to what's in the browsercompany repos.  You're really not that far away from having something perfectly usable.
